inc/Engine/Optimization/RUCSS/Jobs/Manager.php

```diff
@@ -192,7 +189,7 @@ public function set_request_param(): array {
 		 *
 		 * @since 3.14
 		 *
-		 * @param array $skipped_attr Array of safelist values.
+		 * @param array|object $skipped_attr Array of safelist values.
```

Comment:
if we have the conditional after that only match with an array, how can the param be an object?